include/DataFile.h

```diff
@@ -71,7 +72,9 @@ class LMMS_EXPORT DataFile : public QDomDocument
 	QString nameWithExtension( const QString& fn ) const;
 
 	void write( QTextStream& strm );
-	bool writeFile( const QString& fn );
+	bool writeFile(const QString& fn, bool withResources = false);
+	bool copyResources(const QString& resourcesDir);
```

Comment:
Maybe it should be explicitly stated that copyResources can modify the attributes (if files have the same name)? I know that when `Song` loads something, the `DataFile` is discarded anyways, so it's no problem, but for future coders, it might not be obvious that this changes the XML content.

Reply:
Added a small Doxygen comment that should warn about the DataFile will be modified.